### DISCUSSION TITLE
I have a program which does the 2D detector intégration and I want to use the result in MAUD

### DISCUSSION THREAD

**Discussion by picca** [lanl/MILK#56] [General]
Hello, at SOLEIL we are trying to use MAUD in order to plot pole figures on the MARS beamline.

We are using binoculars-ng (https://repo.or.cz/hkl) in order to do the 2D detector integration.

This program generate an hdf5 file which contains a matrix with 2 axis (tth and the sample rotation). each cell of this matrix contain 2 informations, the voxel intensity and the number of detector pixel used to build this voxel.

I would like to know how to generate an input file for MAUD from our data files.

thanks for your help.

Frederic

**Comment by djm87** [comment]
Hi Frederic, 

You can take a look at this [example](https://github.com/lanl/MILK/blob/cec74de9f9046868dc543e82b2916389d8dcfbd9/MILK/integration/integrate.py#L516) of the esg format that MAUD supports. You can append as many esg file headers/data as needed to describe your spectra and sample rotations (presumably omega) in this case. If omega is not the right rotation, you might rotate the sample frame in MAUD or specify e.g. a chi or phi rotation. 

Dan
